### ISSUE TITLE
Why dbm-gorm-diff add unused setTableRemarks and recreate index ?

### ISSUE THREAD

**Issue by abcfy2** [grails/grails-database-migration#185]
If you are unsure about something and the issue is more of a question a better place to ask questions is on Stack Overflow (http://stackoverflow.com/tags/grails) or Slack (http://slack-signup.grails.org). DO NOT use the issue tracker to ask questions.

### Task List

- [ ] Steps to reproduce provided
- [ ] Stacktrace (if present) provided
- [ ] Example that reproduces the problem uploaded to Github
- [ ] Full description of the issue provided (see below)

### Steps to Reproduce

1. ./grails dbm-gorm-diff

### Expected Behaviour

Without unused charsets.

### Actual Behaviour

Sample result:

```groovy
// So many unused setTableRemarks change set
    changeSet(author: "fengyu (generated)", id: "1607565763363-1") {
        setTableRemarks(tableName: "xxx1")
    }

    changeSet(author: "fengyu (generated)", id: "1607565763363-2") {
        setTableRemarks(tableName: "xxx2")
    }

    changeSet(author: "fengyu (generated)", id: "1607565763363-3") {
        setTableRemarks(tableName: "xxx3")
    }
//....
// So many dropIndex and createIndex
    changeSet(author: "fengyu (generated)", id: "1607565763363-24") {
        dropIndex(indexName: "idx1", tableName: "table1")

        createIndex(indexName: "idx1", tableName: "table1", unique: "false") {
            column(name: "column1")
        }
    }

    changeSet(author: "fengyu (generated)", id: "1607565763363-25") {
        dropIndex(indexName: "idx2", tableName: "table2")

        createIndex(indexName: "idx2", tableName: "table2", unique: "false") {
            column(name: "column2")
        }
    }
// ...
```

### Environment Information

- **Operating System**: Linux
- **Grails Version:** 4.0.5
- **Plugin Version:** 3.1.0
- **Database:** PostgreSQL
- **JDK Version:** 1.8

**Comment by sudewa** [comment]
I also had the same issue where I get unnecessary dropIndex and create index.

Database is MySSQL.
plugin : database-migration:3.1.0.RC1

**Comment by robertoschwald** [comment]
This is a Liquibase problem which was fixed recently with version 3.10.1.
You can add   `compile 'org.liquibase:liquibase-core:3.10.1'` to build.gradle dependencies for now to upgrade Liquibase until this plugin was upgraded.

**Comment by abcfy2** [comment]
OK. Thanks.
